src/contentmanager.cpp

```diff
@@ -31,6 +32,9 @@ ContentManager::ContentManager(Library* library, kiwix::Downloader* downloader,
     mp_view->setModel(managerModel);
     mp_view->show();
 
+    auto managerDelegate = new ContentManagerDelegate();
+    mp_view->setItemDelegate(managerDelegate);
+
```

Comment:
> Could we use the ContentManagerDelegate to draw the description instead of relying on a sub node ?

I don't believe this is possible, the model needs to know all data and nodes